scripts/utilities/update_revision.sh

```diff
@@ -78,13 +84,30 @@ source ${1};
 
 # If ATSRC_PACKAGE_CO is not defined, we skip the revision check.
 if ! [ -n "${ATSRC_PACKAGE_CO}" ] || ! [ -n "${ATSRC_PACKAGE_REV}" ]; then
-	return 0;
+	exit 0;
 fi
 
-# This function print a message with identation
+if [[ ${service} = "github" ]];
+then
+	if [[ "${GITHUB_SIGNATURE}" = "" ]];
```

Comment:
I'm afraid this kind of comparisons only works on newer GNU Bash versions.  Usually, you can do the same thing using `-z "${GITHUB_SIGNATURE}"`.